SparkPost/2web2ui
# FE-869 - Adds CodeBlock wrapper and supporting unit test

[FE-869](https://jira.int.messagesystems.com/browse/FE-869)

### What Changed
- Adds `CodeBlock` Matchbox wrapper
- Adds supporting unit tests

### How To Test
- Navigation to `/inbox-placement/details/<id>/content` and see the `<CodeBlock/>` in action. Verify that the component renders in both 

### To Do
